libkbfs/chat_rpc.go

```diff
@@ -421,15 +428,24 @@ func (c *ChatRPC) GetGroupedInbox(
 			continue
 		}
 
-		// TODO: ignore TLFs that aren't in your favorites list.
-
 		tlfType := tlf.Private
 		if info.Visibility == keybase1.TLFVisibility_PUBLIC {
 			tlfType = tlf.Public
 		} else if info.MembersType == chat1.ConversationMembersType_TEAM {
 			tlfType = tlf.SingleTeam
 		}
 
+		tlfIsFavorite := false
+		for _, fav := range favorites {
```

Comment:
Probably best to build a map of this once right after fetching favorites, so we don't have to iterate it every time.  Something like `map[Favorite]bool`.